galleon-pack/galleon-shared/src/main/resources/modules/system/layers/base/io/opentelemetry/api/main/module.xml

```diff
@@ -10,15 +10,9 @@
     <resources>
         <artifact name="${io.opentelemetry:opentelemetry-api}"/>
         <artifact name="${io.opentelemetry:opentelemetry-api-incubator}"/>
-        <artifact name="${io.opentelemetry.instrumentation:opentelemetry-instrumentation-api}"/>
-        <artifact name="${io.opentelemetry.instrumentation:opentelemetry-instrumentation-api-incubator}"/>
-        <artifact name="${io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations}"/>
-        <artifact name="${io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations-support}"/>
-        <artifact name="${io.opentelemetry.instrumentation:opentelemetry-runtime-telemetry-java8}"/>
```

Comment:
Note: this affects what resources (classes etc) are visible to deployments via OpenTelemetryDependencyProcessor and MicroProfileTelemetryDependencyProcessor, as the lists of modules those add to deployments don't include all the ones that have these jars. Any concerns there?

It may very well be a good thing to not be exposing these any more; I have no idea.

Reply:
Right. See the comment above. They're made available via the other/correct module they're included in.

Reply:
Doh! Thanks.